src/main/java/at/hannibal2/skyhanni/test/SkyHanniDebugsAndTests.kt

```diff
@@ -520,6 +523,41 @@ class SkyHanniDebugsAndTests {
         config.debugPos.renderStringsAndItems(displayList, posLabel = "Test Display")
     }
 
+    @SubscribeEvent
+    fun onGuiRenderChestGuiOverlayRender(event: GuiRenderEvent.ChestGuiOverlayRenderEvent) {
+        @Suppress("ConstantConditionIf")
+        if (false) {
+            dragAbleTest()
+        }
+    }
+
+    private fun dragAbleTest() {
+        val bone = ItemStack(Items.bone, 1).toDragItem()
+        val leave = ItemStack(Blocks.leaves, 1).toDragItem()
+
+        config.debugItemPos.renderRenderables(
+            listOf(
+                DragNDrop.dragAble(Renderable.string("A Bone"), { bone }),
+                Renderable.placeholder(0, 30),
+                DragNDrop.dragAble(Renderable.string("A Leave"), { leave }),
```

Comment:
this is very minor since this is just debug, but I think you mean a "leaf"

Reply:
True, I can't spell